code/game/machinery/requests_console.dm

```diff
@@ -375,6 +406,11 @@ var/list/obj/machinery/requests_console/allConsoles = list()
 				icon_state="req_comp_rewired"
 	if (istype(O, /obj/item/weapon/screwdriver))
 		if(open)
+			if(prob(50))

```

Comment:
I think the original had a shock chance, but if you don't like it I'll take it out.